app/services/stats/base_monthly_performance_report.rb

```diff
@@ -2,12 +2,14 @@ module Stats
   class BaseMonthlyPerformanceReport < BaseReport
 
     R005_SPECIFIC_COLUMNS = {
-        month:    'Month'
-    }
+      month:    'Month'
+    }.freeze
 
     R005_SPECIFIC_SUPERHEADINGS = {
-        month:     ''
-    }
+      month:     ''
+    }.freeze
+
+    INDEXES_FOR_PERCENTAGE_COLUMNS = [1, 7, 13].freeze
 
```

Comment:
nice.  I think there is a rubocop rule that you can add which will insist on all constants being frozen.